From 9e596feb28e25b31dbdff6bd62bdb050865447f2 Mon Sep 17 00:00:00 2001
From: Sean Hoyt <seanhoyt963@gmail.com>
Date: Sat, 14 Jul 2018 06:03:15 +0200
Subject: [PATCH] Add /proc/ged to zygote whitelist

Change-Id: I07275d0d38c22e8c743bf3ec91e88959a4c32540
---

diff --git a/core/jni/fd_utils.cpp b/core/jni/fd_utils.cpp
index 92a5426..cb7384d 100644
--- a/core/jni/fd_utils.cpp
+++ b/core/jni/fd_utils.cpp
@@ -42,6 +42,7 @@
   "/dev/urandom",
   "/dev/ion",
   "/dev/dri/renderD129", // Fixes b/31172436
+  "/proc/ged",
   "/system/framework/org.lineageos.platform-res.apk",
 };
 
